pkg/storage/buckets/store.go

```diff
@@ -132,7 +132,7 @@ func (b *BucketStore) List(ctx context.Context, startAfter, endBefore string, li
 			Meta:   convertMeta(itm.Meta),
 		})
 	}
-	return items, more, nil
+	return items, more, err
```

Comment:
Perhaps we can change this line back and do multiple function exit points with something like
```
objItems, more, err := b.o.List(ctx, nil, paths.New(startAfter), paths.New(endBefore), false, limit, meta.Modified)
if err != nil {
return ...
}
```